java/cuvs-java/src/main/java22/com/nvidia/cuvs/spi/JDKProvider.java

```diff
@@ -31,6 +31,10 @@
 
 final class JDKProvider implements CuVSProvider {
 
+  static {
+    OptionalNativeDependencyLoader.loadLibraries();
```

Comment:
This is the call into the optional native-dependency loader.  

If the jar includes native dependency libraries, they will be loaded at startup.  If not, the load is skipped, and it runs as before (i.e. depending on `$LD_LIBRARY_PATH`, etc.)